### PR TITLE
Printing style for current page link in header

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
Changing the blue back to the brand colour `#1d70b8` when printing.  The colour `#1d8feb` is presumably to contrast against the black `#0b0c0c` header, but when printing the header is white `#ffffff`, so it makes sense to use a darker blue which contrasts better against white.

### CODE REVIEW

**Review by querkmachine — APPROVED** [review]
Does exactly what it says on the tin.

Needs a second approval before merging. Probably warrants a changelog entry too?

**Review by owenatgov — COMMENTED** [review]
@MalcolmVonMoJ Looks good to me too 👍🏻 

Before I give this the official github tick, I'd like to ask that you add a changelog entry and squash the commit history. We're happy to handle this if you prefer.

**Comment by MalcolmVonMoJ** [comment]
@owenatgov Can you do this please?  I don't know how to squash in github.  I went to add it to the changelog too, but mistook "feature release" for "future release" and put it in the wrong place.  Undid that error, but I think someone else is best to squash and update the changelog as I'm not sure where it is.  

Also, one of the automated tests was failing and I am not sure what to do about it.

**Review by owenatgov — APPROVED** [review]
Approving as the code is 👌🏻 I'll tidy up the commit history, add a changelog entry and merge in a jiffy.

**Pull request merged**
